build/configs/rtl8720e/hello/defconfig

```diff
@@ -258,7 +258,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 # Board-Partition Options
 #
-#CONFIG_FLASH_PARTITION=y
+CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="32,28,16,4,432,3068,3068,512,1024,8,"
 CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,none,bootparam,"
```

Comment:
could you rename ss_reserved to reserved here and tc as well?

Reply:
done